src/HomeHeader.jsx

```diff
@@ -0,0 +1,67 @@
+import "./homeHeader.css"
+import Logo from "./static/images/logo.jpg"
+import Restaurant from "./static/images/restaurant.png"
+import ArrowDown from "./static/images/arrow-down.svg"
+import Search from "./static/images/search.svg"
+import Pin from "./static/images/pin-o.svg"
+import Promotion from "./static/images/promotion.svg"
+import User from "./static/images/user.svg"
+import Bag from "./static/images/bag.svg"
+
+function HomeHeader(){
+  return(
+    <header className="home_header">
+        <nav class="home-header__container">
+            <div class="home-header__logo">
+              <img src={Logo} width="100" height="100" />
+            </div>
+            <section class="multi-categories-selector__label">
+                <div class="multi-categories-selector__background-colors-restaurant">
+                    <img src={Restaurant} width="40" height="40" />
+                </div>
+                <div class="multi-categories-selector__label-title">
+                    <span>Restaurante</span>
+                </div>
+                <div class="multi-categories-selector__label-arrow">
+                    <span class="icon-marmita--arrow-down">
```

Comment:
Gi, this classNames are the website project, as this class `multi-categories-selector`, I think this is not a good idea, because in this project you can practice the bests practices, like the BEM.  I suggestion that you not copy from the website 🚀